### PR TITLE
[WIP] Creating a Fast RTPS Listener application Example updated

### PR DESCRIPTION
**Pull request by amano001**
Fixed example to reflect changes to the sensor_combined_ file and example code. It does not include magnotometer and barometer in the functions and therefore code will not build. As well as now functions have trailing _.

### CODE REVIEW

**Comment by hamishwillee** [comment]
Thanks @amano001. @TSC21 Are you good with this? If so, hit merge :-)

File: en/middleware/micrortps.md [hunk 1]
```diff
@@ -421,23 +421,16 @@ void sensor_combined_Subscriber::SubListener::onNewDataMessage(Subscriber* sub)
...
+            std::cout << "gyro_rad: " << st.gyro_rad_().at(0);
+            std::cout << ", " << st.gyro_rad_().at(1);
+            std::cout << ", " << st.gyro_rad_().at(2) << std::endl;
+            std::cout << "gyro_integral_dt: " << st.gyro_integral_dt_() << std::endl;
+            std::cout << "accelerometer_timestamp_relative: " << st.accelerometer_timestamp_relative_() << std::endl;
+            std::cout << "accelerometer_m_s2: " << st.accelerometer_m_s2_().at(0);
+            std::cout << ", " << st.accelerometer_m_s2_().at(1);
+            std::cout << ", " << st.accelerometer_m_s2_().at(2) << std::endl;
+            std::cout << "accelerometer_integral_dt: " << st.accelerometer_integral_dt_() << std::endl;
+
```
Comment (TSC21) [inline en/middleware/micrortps.md:433]: Please also add the `accelerometer_clipping`: https://github.com/PX4/Firmware/blob/master/msg/sensor_combined.msg#L20.

File: en/middleware/micrortps.md [hunk 2]
```diff
@@ -400,7 +400,7 @@ Enter the following commands to create the application:
 cd /path/to/PX4/Firmware/build/px4_sitl_rtps/src/modules/micrortps_bridge
 mkdir micrortps_listener
 cd micrortps_listener
-fastrtpsgen -example x64Linux2.6gcc ../micrortps_client/micrortps_agent/idl/sensor_combined.idl
+fastrtpsgen -example x64Linux2.6gcc ../micrortps_client/micrortps_agent/idl/sensor_combined_.idl
```
Comment (TSC21) [inline en/middleware/micrortps.md:403]: This isn't correct. I just generated the agent using `make px4_sitl_rtps` and the files do not have trailing underscores:
```sh
├── idl
│   ├── adc_report.idl
│   ├── airspeed.idl
│   ├── battery_status.idl
│   ├── camera_capture.idl
│   ├── camera_trigger.idl
│   ├── collision_constraints.idl
│   ├── collision_report.idl
│   ├── cpuload.idl
│   ├── debug_array.idl
│   ├── debug_key_value.idl
│   ├── debug_value.idl
│   ├── debug_vect.idl
│   ├── distance_sensor.idl
│   ├── estimator_status.idl
│   ├── home_position.idl
│   ├── iridiumsbd_status.idl
│   ├── obstacle_distance.idl
│   ├── onboard_computer_status.idl
│   ├── optical_flow.idl
│   ├── position_setpoint.idl
│   ├── position_setpoint_triplet.idl
│   ├── radio_status.idl
│   ├── satellite_info.idl
│   ├── sensor_baro.idl
│   ├── sensor_combined.idl
│   ├── sensor_selection.idl
│   ├── timesync.idl
│   ├── trajectory_waypoint.idl
│   ├── vehicle_attitude.idl
│   ├── vehicle_mocap_odometry.idl
│   ├── vehicle_odometry.idl
│   ├── vehicle_trajectory_waypoint.idl
│   ├── vehicle_visual_odometry.idl
│   ├── vtol_vehicle_status.idl
│   └── wind_estimate.idl
```
You might have a problem with your FastRTPS installation where it is finding a version you have installed that is equal or lower than 1.7.2 - https://github.com/PX4/Firmware/blob/master/msg/tools/px_generate_uorb_topic_files.py#L200-L202. Please check on your system for the `libfastrps` links using `ldconfig -v | grep libfastrtps`, and make sure that you only have 1.8.0 or above.

**Review by TSC21 — CHANGES_REQUESTED** [review]

File: en/middleware/micrortps.md (hunk 2 above)
Comment (amano001) [reply en/middleware/micrortps.md:403]: @TSC21 
looks like I have 1.8.3 installed.  

/sbin/ldconfig.real: Can't stat /usr/local/lib/x86_64-linux-gnu: No such file or directory
/sbin/ldconfig.real: Path `/lib/x86_64-linux-gnu' given more than once
/sbin/ldconfig.real: Path `/usr/lib/x86_64-linux-gnu' given more than once
/sbin/ldconfig.real: /lib/x86_64-linux-gnu/ld-2.27.so is the dynamic linker, ignoring

	libfastrtps.so.1 -> libfastrtps.so.1.8.3
/sbin/ldconfig.real: /lib32/ld-2.27.so is the dynamic linker, ignoring

Comment (TSC21) [reply en/middleware/micrortps.md:403]: What's the output of `fastrtpsgen -version`?

Comment (amano001) [reply en/middleware/micrortps.md:403]: @TSC21 
fastrtpsgen -version
openjdk version "11.0.6" 2020-01-14
OpenJDK Runtime Environment (build 11.0.6+10-post-Ubuntu-1ubuntu118.04.1)
OpenJDK 64-Bit Server VM (build 11.0.6+10-post-Ubuntu-1ubuntu118.04.1, mixed mode, sharing)
fastrtpsgen version 1.0.4

Comment (amano001) [reply en/middleware/micrortps.md:403]: @TSC21 Hey have you been able to see why with my versions it might not have the same files you are building?

**Comment by hamishwillee** [comment]
@TSC21 @amano001 What is the status of this? What has to happen for it to go in (or be closed)?

File: en/middleware/micrortps.md (hunk 2 above)
Comment (TSC21) [reply en/middleware/micrortps.md:403]: The generated IDL's terminate without `_`. So, not applicable.

**Comment by TSC21** [comment]
Closing this for now. I will update the output of the sensor_combined message in a later PR.

**Comment by amano001** [comment]
@TSC21 Sorry for not responding, I had dropped off a little bit when I had to transfer schools.  I think closing is good for now, I will revisit reinstalling everything once I get settled in again and comment if I run into anything new.

**Pull request closed without merging**